docs/_posts/examples/v1.0.0/0100-01-01-featurelayer-to-pointtolayer.html

```diff
@@ -0,0 +1,62 @@
+---
+layout: example
+categories: example/v1.0.0
+version: v1.0.0
+title: Marker styles using pointToLayer
+description: Set marker style in a featureLayer with pointToLayer.
+tags:
+  - markers
+  - featureLayer
+  - style
+---
+
+<div id='map'></div>
+<script>
+var map = L.mapbox.map('map', 'examples.h186knp8')
+    .setView([45.52245801087795, -122.67773866653444], 3);
+    
+var geojson = {
+  "type": "FeatureCollection",
+  "features" : [{
+  "type": "Feature",
+    "geometry": { 
+        "type": "Point", 
+        "coordinates": [-120.086,37.857]
+     }, 
+    "properties": {
+      "title" : "AREA 10",
+      "radius" : 10
+     }
+},{
```

Comment:
Will the spacing normalize when publish or do we have to take care of that here? If so, make sure spacing is correct.